scripts/singletons/jdungeon.gd

```diff
@@ -177,12 +177,18 @@ func register_class_resource(charclass_class: String, class_res_path: String):
 
 ## There'll be a LOT of dialogue, so loading each manually could not only dramatically increase load times, but it could also make errors more likely.
 func register_dialogue_all(dialogue_folder: String):
-	for file: String in DirAccess.get_files_at(dialogue_folder):
+	var allFiles: PackedStringArray = DirAccess.get_files_at(dialogue_folder)
```

Comment:
I am pretty sure the problem is with this function.

This function will look at the files located at "res://scenes/ui/dialogue/Dialogues/"

However this directory location does not exist in a exported build.

So there are 2 options here:

1) make sure this directory is present in the exported builds
2) go for KISS and load the res

Reply:
You're sure that not even the .import files exist?  
If not, then i'll just have to embrace the KISS